### ISSUE TITLE
Support ! in .capstanignore

### ISSUE THREAD

**Issue by miha-plesko** [mikelangelo-project/capstan#92]
See this comment https://github.com/mikelangelo-project/capstan/issues/91#issuecomment-362343678 where @aitorhh requested such feature.

It wouldn't be too much of a trouble to at least support very basic functionality (without any wildcards):

```
# explicitly include specific file (must not contain *)
!/path/to/file.txt

# explicitly include whole directory and its content (must not contain *)
!/path/to/directory
```

@aitorhh would this be sufficient for your usecase?

**Comment by aitorhh** [comment]
Yes, I think allows the users to be more selective with the files that want to include. And allows a broader set of project structures to be used :)

Thanks!
